Scripts/Scenes/Lobby/SceneLobby.cs

```diff
@@ -18,6 +14,8 @@ public class SceneLobby : AScene
         [Export] public readonly NodePath NodePathBtnReady;
         [Export] public readonly NodePath NodePathBtnStart;
 
+        public LobbyNotifier LobbyNotifier;
```

Comment:
Perhaps this should be `private LobbyNotifier _lobbyNotifier;`?

https://stackoverflow.com/questions/295104/what-is-the-difference-between-a-field-and-a-property